openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -786,6 +797,13 @@ public final void clearFieldsInternal(T entity, Fields fields) {
     clearFields(entity, fields);
   }
 
+  public final void clearSearchFieldsInternal(Map<String, Object> json) {
+    json.remove("fqnParts");
```

Comment:
btw this can be achieved on the elastic search request itself, ie we can make call to ES/OS with fields we need